hls4ml/converters/keras_to_hls.py

```diff
@@ -124,7 +124,7 @@ def parse_default_keras_layer(keras_layer, input_names):
     return layer
 
 def parse_data_format(input_shape, data_format='channels_last'):
-    if input_shape[0] is None:
+    if len(input_shape) == 4 or input_shape[0] is None:
```

Comment:
I think we can simply remove this check altogether. The first dimension is always the batch size, and should always be discarded. Right @vloncar ?